ush/load_ufsda_modules.sh

```diff
@@ -34,13 +34,13 @@ source "${HOMEgfs}/ush/module-setup.sh"
 module use "${HOMEgfs}/sorc/gdas.cd/modulefiles"
 
 case "${MACHINE_ID}" in
-  ("hera" | "orion" | "hercules")
+  ("hera" | "orion" | "hercules" | "wcoss2")
     module load "${MODS}/${MACHINE_ID}"
     ncdump=$( command -v ncdump )
     NETCDF=$( echo "${ncdump}" | cut -d " " -f 3 )
     export NETCDF
     ;;
-  ("wcoss2" | "acorn" | "jet" | "gaea" | "s4")
+  ("jet" | "gaea" | "s4")
```

Comment:
Since "acorn" was removed from Line 37, should it be added back here?

Reply:
Good catch @CatherineThomas-NOAA .  `acorn` added back at [`7ebfdde`](https://github.com/NOAA-EMC/global-workflow/pull/2620/commits/7ebfdde398542fb3648b1665b3937326a4a25b17)